tests/fixture_plugins/observations_fixtures/observations_api_fixtures.py

```diff
@@ -80,3 +81,36 @@ def mip_dna_observations_api(
     mocker.patch.object(LimsAPI, "get_source", return_value=SourceType.TISSUE)
 
     return mip_dna_observations_api
+
+
+@pytest.fixture
+def raredisease_observations_api(
+    cg_context: CGConfig,
+    analysis_store: Store,
+    lims_api: LimsAPI,
+    loqusdb_api: LoqusdbAPI,
+    case_id: str,
+    customer: Customer,
+    number_of_loaded_variants: int,
+    loqusdb_id: str,
+    mocker: MockFixture,
+) -> RarediseaseObservationsAPI:
+    """Rare diseases observations API fixture."""
+    raredisease_observations_api = RarediseaseObservationsAPI(cg_context)
+    raredisease_observations_api.store = analysis_store
+    raredisease_observations_api.loqusdb_api = loqusdb_api
+
+    # Mocked case scenario for RAREDISEASE uploads
```

Comment:
```suggestion
    # Mocked case scenario for RAREDISEASE uploads
```
Could you update this to describe what is set instead?